.github/workflows/lint-test.yaml

```diff
@@ -61,6 +61,16 @@ jobs:
       - run: yarn lint
         working-directory: developer-docs-site
 
+  rust-cryptohasher-domain-separation-check:
+    runs-on: high-perf-docker
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3
+      - uses: actions/setup-python@13ae5bb136fac2878aff31522b9efb785519f984
+        with:
+          python-version: '3.10'
```

Comment:
do you actually need 3.10?

I believe we have a .python file that sets 3.9 for all other deps currently

so you can remove this